testflinger_agent/client.py

```diff
@@ -55,6 +58,28 @@ def _requests_retry(self, retries=3):
         session.mount("https://", adapter)
         return session
 
+    def _configure_influx(self):
+        """Configure InfluxDB client using environment variables.
+
+        :return: InfluxDBClient object
+        """
+        # required influxdb env vars
+        req_env_vars = ["INFLUX_HOST", "INFLUX_USER", "INFLUX_PW"]
+
+        if not all(var in os.environ for var in req_env_vars):
+            return
```

Comment:
So the docstring is not accurate... we're either returning nothing, or we are returning an InfluxDBClient object.  Instead of returning two possible types, I would recommend just having this method set self.influx_client to the object.

Reply:
Played around with this, since it's None and an object type, that should be pythonic IIRC.
I'm not married to it though and can define self.influx_client = influx_client.

this version looks a bit cleaner, lmk thoughts...